### PR TITLE
Remove testbed from unit tests

### LINKED ISSUE #18: Remove testbed from unit tests

### PR DESCRIPTION
**Pull request by DanielMuckelbauer**
closes #18

### CODE REVIEW

File: src/app/home/home.component.spec.ts [hunk 1]
```diff
@@ -1,33 +1,25 @@
...
+    dialogSpy = spyOn(dialog, 'open').and.returnValue(dialogRefSpyObj);
 
-    dialogSpy = spyOn(TestBed.get(MatDialog), 'open').and.returnValue(dialogRefSpyObj);
+    component = new HomeComponent(dialog);
   });
 
   it('opens the dialog with configuration', () => {
     component.openDialog();
 
-    expect(dialogSpy).toHaveBeenCalledWith(RoomCreationModalComponent, { width: "20rem" });
```
Comment (Hakem22) [inline src/app/home/home.component.spec.ts:31]: Maybe a lint rule?
image

File: src/app/app.component.spec.ts [hunk 2]
```diff
@@ -1,28 +1,13 @@
...
-    await TestBed.configureTestingModule({
-      imports: [
-        RouterTestingModule
-      ],
-      declarations: [
-        AppComponent
-      ],
-    }).compileComponents();
-  });
+  let component: AppComponent;
```
Comment (Hakem22) [inline src/app/app.component.spec.ts:4]: What about firebase-room-service, you racist!

**Review by Hakem22 — APPROVED** [review]

File: src/app/app.component.spec.ts (hunk 2 above)
Comment (DanielMuckelbauer) [reply src/app/app.component.spec.ts:4]: That service did not exist when I created the PR!

File: src/app/home/home.component.spec.ts (hunk 1 above)
Comment (DanielMuckelbauer) [reply src/app/home/home.component.spec.ts:31]: I don't get it, what is the problem here?

Comment (DanielMuckelbauer) [reply src/app/home/home.component.spec.ts:31]: Maybe you are

Comment (Hakem22) [reply src/app/home/home.component.spec.ts:31]: Add a rule that makes the use of `"` fails the lint, so we will be forced to use `'`

File: src/app/app.component.spec.ts (hunk 2 above)
Comment (Hakem22) [reply src/app/app.component.spec.ts:4]: Yeah right, I am gonna act like I believe you

File: src/app/home/home.component.spec.ts (hunk 1 above)
Comment (DanielMuckelbauer) [reply src/app/home/home.component.spec.ts:31]: Ignore as discussed :) It is already configured correctly, just CI doesn't run lint it seems.

**Pull request merged**